### PR TITLE
QA-336 Create the pr_ branches with exceptions

### PR DESCRIPTION
**Pull request by merlin-northern**
ChangeLog:title
Signed-off-by: Peter Grzybowski <peter@northern.tech>

### CODE REVIEW

**Comment by merlin-northern** [comment]
Kristian: if you would be so kind and give me an early review here, is this the right direction? before I start adding tests.

**Comment by merlin-northern** [comment]
> Kristian: if you would be so kind and give me an early review here, is this the right direction? before I start adding tests.

@kacf thanks for early feedback over slack. I hope it makes sense. do you think we need more tests around it?

File: main_pullrequest.go [hunk 1]
```diff
@@ -11,8 +12,32 @@ import (
 	clientgithub "github.com/mendersoftware/integration-test-runner/client/github"
 )
 
-func processGitHubPullRequest(ctx *gin.Context, pr *github.PullRequestEvent, githubClient clientgithub.Client, conf *config) error {
+func startPipeline(branchName string) bool {
```
Comment (kacf) [inline main_pullrequest.go:15]: Suggest renaming to `shouldStartPipeline`. The original name makes it sound like you are starting the pipeline (and this is what I first thought when I read it out of context below).

**Review by kacf — APPROVED** [review]
Just a minor suggestion, it looks good to me!

**Pull request merged**